clang/test/lab1/prokofev_kirill/test.cpp

```diff
@@ -0,0 +1,71 @@
+// RUN: split-file %s %t
+// RUN: %clang_cc1 -load %llvmshlibdir/ProkofevRenamePlugin%pluginext -add-plugin rename\
+// RUN: -plugin-arg-rename type=class\
+// RUN: -plugin-arg-rename oldName=Alpha\
+// RUN: -plugin-arg-rename newName=Beta %t/class_test.cpp
+// RUN: FileCheck %s < %t/class_test.cpp --check-prefix=CLASS
+
+// CLASS: class Beta {
+// CLASS-NEXT: public:
+// CLASS-NEXT:  Beta(){};
+// CLASS-NEXT:  ~Beta();
+// CLASS-NEXT:};
+
+//--- class_test.cpp
+class Alpha {
+public:
+  Alpha(){};
+  ~Alpha();
+};
```

Comment:
Please, extend test suite with more cases: function definitions outside the class, variables that have "Alpha"  as a substring in their names and so on

Reply:
Updated the class test